Common/src/Common/Service/Table/Formatter/InternalConversationMessage.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Internal conversation message
+ */
+
+namespace Common\Service\Table\Formatter;
+/**
+ * Internal conversation message
+ */
+class InternalConversationMessage implements FormatterPluginManagerInterface
+{
+    /**
+     * status
+     *
+     * @param array $row    Row data
+     * @param array $column Column data
+     *
+     * @return     string
+     * @inheritdoc
+     */
+    public function format($row, $column = null)
```

Comment:
Would be nice to have types (and return type) here.

Reply:
Would require a union type for $row like:
`array|String`

But we're still on php 7

Reply:
$row can be a string? It looks like it's expected to always be an array in the code?